lexicons/com/atproto/temp/upgradeRepoVersion.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.temp.upgradeRepoVersion",
```

Comment:
not `com.atproto.unspecced.*`? not a big deal or worth changing, just surprised.

Reply:
The proximate reason is that our lexgen code doesn't had some difficulty with the `unspecced` namespace being defined under both `com.atproto` & `app.bsky`

The post-hoc rationale is that `unspecced` is normally a proof of concept for some feature that we do plan to introduce. This route should only exist for a day or two.